### PR TITLE
Add oath services to examples

### PR DESCRIPTION
**Pull request by tsemana**
@pauldowman - added the user services section, and deployed iron and flow versions to meteor.com

### CODE REVIEW

File: examples/iron-router/iron-router.html [hunk 1]
```diff
@@ -21,6 +21,18 @@
     {{currentIdentity}}
   </div>
   <div>
+    <h3>Current Identity</h3>
+    {{currentIdentity}}
```
Comment (benstr) [inline examples/iron-router/iron-router.html]: Looks like {{currentIdentity}} is shown twice in this template.

**Comment by benstr** [comment]
Hey @tsemana nice! When trying to use the Facebook login I get a "in development mode" error and causes template errors then. I think that is a simple boolean to flip on your FB developer console. Also I think the Github integration should be under OKG's org account. It is easy to do from Github... just need to transfer the integration to OKG.

**Comment by tsemana** [comment]
@benstr - dupe currentIdentity removed, Facebook test apps are fixed, and Github integration's moved to OKG's org.

**Comment by benstr** [comment]
@tsemana I did a little tweak to fix a template error that was being shown on the example sites. I do not have access to deploy to those meteor apps so can you make sure the latest versions are deployed? Thanks man!

**Pull request merged**